src/cdk-experimental/column-resize/resizable.ts

```diff
@@ -222,6 +224,7 @@ export abstract class Resizable<HandleComponent extends ResizeOverlayHandle>
   private _showHandleOverlay(): void {
     this._updateOverlayHandleHeight();
     this.overlayRef!.attach(this._createHandlePortal());
+    this.changeDetectorRef.markForCheck();
```

Comment:
Could you add a comment that gives the context on this call?

Reply:
Done.